docs/development/doc_howto.rst

```diff
@@ -150,6 +150,14 @@ If you get a zsh error try using putting to ignore block inside quotes
 
     pytest --doctest-modules "--ignore-glob=*/tests" gammapy
 
+It is also important to check that you have correctly formatted your docstring. An easy way to check this
+is with the following for your specific file, i.e.:
+
+.. code-block:: bash
+
+    pydocstyle gammapy/data/event_list.py
```

Comment:
Maybe this can be mandated for every commit, like we have for `black` or `flake8`?

Reply:
Can you please give more details on this @Astro-Kirsty? So then we cannot enforce following a docstring format? BTW, ruff also integrates with pycodestyle, so we could just use it directly after defining the rules.

Reply:
We have a number of `pydocstyle`s that we do not implement in gammapy and do not want to. For example the two following
- D107 - Missing docstring in __init__
- D104: Missing docstring in public package (often coming from the __init__.py file being empty, which is the case for the 'tests' folders)

So those will always show incorrectly in the pydocstyle. I understand that you can run `pydocstyle` with the `--ignore=<codes>` but it does not work as expected for me. Therefore I am not sure how we should proceed on this. 

It is also important to note that there are a large number of other errors within the docstrings! Some things I can fix, but it will take time to go over this and make some decision of how to proceed. I am open to ideas!

Reply:
OR contradicting errors!
`D213: Multi-line docstring summary should start at the second line`
Fix this and we receive:
`D212: Multi-line docstring summary should start at the first line`

Reply:
I see. Many thanks for the explanation. Then, I'd say we should follow some of those rules and ignore others in the pydocstyle configuration.

Reply:
Maybe I should also add a note here about that?

Reply:
I think it would be helpful yes. Something like we follow these particular pydocstyle rules.

Reply:
So we tend to follow the numpydoc style, but I do not know if there is a command to check for that as cleanly as the pydocstyle command works